firefox_puppeteer/docs/index.rst

```diff
@@ -23,16 +23,17 @@ along with instructions for installation and usage.
 There are plans to move it alongside the `Marionette python client`_.
 
 .. _Marionette python client: http://marionette-client.readthedocs.org/en/latest/
-.. _firefox-ui-tests: https://github.com/mozilla/firefox-ui-tests/tree/master/firefox_puppeteer
+.. _firefox-ui-tests: https://github.com/mozilla/firefox-ui-tests/tree/mozilla-central/firefox_puppeteer
+
 
 Libraries
 ---------
 
-
 The following libraries are currently implemented. More will be added in the
 future. Each library is available from an instance of the FirefoxTestCase class.
 
 .. toctree::
+
```

Comment:
@whimboo, one of the changes to index.rst makes the output prettier, but the extra line in the toctree was an attempt to fix the auto-generated docs. However, they are still missing from Read The Docs, so I guess it didn't work. Should I file a new bug for this issue?